src/components/account/ResetPassword.test.tsx

```diff
@@ -0,0 +1,48 @@
+import UserService from '@services/user.service'
+import {cleanup, render, screen, waitFor} from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import {REQUEST_RESET_PASSWORD_MESSAGE} from '__test_utils/test_server/userHandlers'
+import {createWrapper} from '__test_utils/utils'
+import {ResetPassword} from './ResetPassword'
+
+export const setUp = () => {
+  const user = userEvent.setup()
+  const spyOnRequestResetPassword = jest.spyOn(UserService, 'requestResetPassword')
+  const element = render(<ResetPassword />, {wrapper: createWrapper()})
+
+  const emailField = screen.getByLabelText('Email')
+  const submitButton = screen.getByRole('button', {name: /reset my password/i})
+
+  return {user, spyOnRequestResetPassword, element, emailField, submitButton}
+}
+
+afterEach(() => {
+  jest.restoreAllMocks()
+  cleanup()
+})
+
+describe.only('ResetPassword', () => {
```

Comment:
Just curious, why `describe.only`? Guessing maybe it's a convention to enforce only one top-level `describe` per file?

Reply:
Oh good catch! This is a holdover from writing the tests, not a convention of the repo -- I'll remove it!